src/main/java/com/derongan/minecraft/mineinabyss/Relic/Relics/RelicType.java

```diff
@@ -41,6 +47,29 @@ default ItemStack getItem() {
         return item;
     }
 
+    default ArmorStand getAndPlaceItem(Location location){
```

Comment:
I'm iffy about having this inside relictype. Maybe we need a new time called PlaceableRelicBehaviour that lets you do this?